Dockerfile

```diff
@@ -1,3 +1,15 @@
-from ruby
+FROM alpine:3.7
 
-run gem install --no-ri --no-rdoc mailcatcher
+RUN apk add --update --no-cache \
+  build-base \
+  curl \
+  ruby-dev \
+  ruby \
+  ruby-bundler \
+  sqlite-dev
+
+RUN gem install --no-ri --no-rdoc json mailcatcher
```

Comment:
we can remove all the dependencies using `from ruby:2.5.1-alpine3.7`, it will increase 40MB to final image, but contains all dependencies we need to run mailcatcher.

Reply:
Don't worry, it is okay this way :)